pkg/kontainer-engine/service/service.go

```diff
@@ -491,6 +493,8 @@ func (r *RunningDriver) Stop() {
 		r.Server.Stop()
 	} else {
 		r.cancel()
+		_ = r.cmd.Wait()
```

Comment:
Nit, but it's a bit more flexible to do this regardless of `r.Builtin` flag and always wait for the command to finish if it's set.
Also, do we have to set it to `nil` as Wait releases resources - https://pkg.go.dev/os/exec#Cmd.Wait ?

Reply:
We can do the first one, but for now there is no `cmd` if `r.Builtin == true`.

I set `r.cmd = nil` so all the memory can be cleaned up not just the resources.

Reply:
We're talking hypothetical case here that if we ever start using `cmd` in other cases the suggested approach will minimize the changes needed. I believe it's best if code follows the "actual logic of the function" and in this case the logic of function to stop RunningDriver is to ensure that `cmd` has exited (if it's set), no matter _why_ `cmd` got set.

Reply:
Done.